### PR TITLE
python3.pkgs.jabberbot: remove

### PR DESCRIPTION
**Pull request by Mic92**
no update since 2016; python2 only due to xmpppy

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Mic92** [comment]
I also thought about remove xmpppy... only pyMAILt and pyIRCT are based on that @7c6f434c added those back in the old days when nixpkgs was still svn. I highly doubt that anyone is still using that.

**Review by davidak — APPROVED** [review]
that is resonable

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3291,8 +3291,6 @@ in {
 
   j2cli = callPackage ../development/python-modules/j2cli { };
 
-  jabberbot = callPackage ../development/python-modules/jabberbot { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:3294]: Why are we not adding aliases for those packages?

Comment (Mic92) [reply pkgs/top-level/python-packages.nix:3294]: no.

**Pull request merged**